airflow/config_templates/default_airflow.cfg

```diff
@@ -122,6 +122,12 @@ security =
 # values at runtime)
 unit_test_mode = False
 
+# User defined logging configuration file path.
+logging_config_path =
```

Comment:
Oh this specifies the filename of the user defined logging configuration if they do not want to use the default `default_airflow_logging.py`. It's like specifying the path for this config file.